src/first_steps/intro.md

```diff
@@ -22,7 +22,7 @@ To be allowed to write files, specify the `-w` option to rizin when opening a fi
 ```
 
 Appending a `?` to a command will show its help message, for example, `p?`.
-Appending `?*` will show commands starting with the given string, e.g. `p?*`.
+Using `?* <command>` will show all commands starting with the given string, e.g. `?* p`.
```

Comment:
Can't you still append `?*` to each command?

Reply:
no. not with newshell. is there a way to tell the shell to provide as argv0 that?